### ISSUE TITLE
[Feature] 纯文本复制 copy as plain text

### ISSUE THREAD

**Issue by SBBTD** [Bin-Huang/chatbox#1119]
**问题描述**
目前手动选中文本后，使用`Ctrl+C`或`右键复制`到word或其他富文本编辑器，会有字体、背景等的格式，
希望能默认复制为`纯文本`，不带格式的

**解决思路**
不太了解项目用的客户端框架，或许可以通过自定义Ctrl+C的回调来实现

**附加上下文**
![image](https://github.com/Bin-Huang/chatbox/assets/26642340/9a5ecaac-4094-45b3-9bb4-1bbeae8e9aaf)

**Comment by jpike88** [comment]
@Bin-Huang agree, copying color of text, formatting etc makes it hard to use with other programs
